### PR TITLE
Assign tickets using case insensitive email.

### LINKED ISSUE #1015: Assigning tickets via email should use lowercase emails
Right now, trying to assign a ticket to `USER@test.com` will fail if user's email is `user@test.com`.

### PR DESCRIPTION
**Pull request by umgelurgel**
fix #1015

### CODE REVIEW

File: tests/test_user_panel.py [hunk 1]
```diff
@@ -23,7 +24,7 @@
 )
 
 from tests.common_tools import make_user
-
+from .test_cart import _setup
```
Comment (artcz) [inline tests/test_user_panel.py]: Not a big fan of this one, maybe it's time to export this _setup to some common module, and call it `def setup_conference_with_typical_fares` or copy the code here and also call it _setup? I'm thinking about avoiding weird, non obvious dependencies, like private functions from another module. WDYT?

File: conference/user_panel.py [hunk 2]
```diff
@@ -181,7 +181,10 @@ def clean_email(self):
         return self.cleaned_data["email"]
 
     def get_user(self):
-        return User.objects.get(email=self.cleaned_data["email"])
+        return User.objects.get(
+            is_active=True,
+            email__iexact=self.cleaned_data["email"],
```
Comment (artcz) [inline conference/user_panel.py:186]: good call, I'd probably use `email=self.cleaned_data['email'].lower()` but I'm not sure all the emails in the db are converted to lowercase (we do that on the registration from, but iirc only started doing this last year)

**Review by artcz — CHANGES_REQUESTED** [review]
Good work, left a comment about _setup, let me know what you think. :)

File: tests/test_user_panel.py (hunk 1 above)
Comment (umgelurgel) [reply tests/test_user_panel.py]: I like the idea of moving it to a common module and renaming it; it makes it clearer what it does :)

File: conference/user_panel.py (hunk 2 above)
Comment (umgelurgel) [reply conference/user_panel.py:186]: Yeah, the `iexact` is probably slower, but makes no assumption about the data on the db.

File: tests/common_tools.py [hunk 3]
```diff
@@ -146,3 +148,20 @@ def is_using_jinja2_template(response):
     res = response.resolve_template(response.template_name)
     assert res.backend.name == "jinja2", res.backed.name
     return True
+
+
+def setup_conference_with_typical_fares(start=date(2019, 7, 8), end=date(2019, 7, 14)):
+    Conference.objects.get_or_create(
+        code=settings.CONFERENCE_CONFERENCE,
+        name=settings.CONFERENCE_NAME,
+        # using 2018 dates
```
Comment (artcz) [inline tests/common_tools.py]: This is probably out of date now. :)

**Review by artcz — APPROVED** [review]
LGTM

File: tests/common_tools.py (hunk 3 above)
Comment (umgelurgel) [reply tests/common_tools.py]: True :)

**Pull request merged**